src/sandstorm/web-session.capnp

```diff
@@ -38,6 +38,11 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
   # application's storage was written and may automatically implement etags based on hashing the
   # content.
 
+  const appHeaderPrefix :Text = "x-sandstorm-app-";
```

Comment:
It feels slightly odd to specify this in lower-case when lower-casing of headers is specifically a property of Node. Perhaps we should specify it as "X-Sandstorm-App-" and expect the caller to lower-case it if the context calls for that?